src/main/java/ro/unibuc/hello/controller/UserController.java

```diff
@@ -0,0 +1,58 @@
+package ro.unibuc.hello.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+import ro.unibuc.hello.data.User.User;
+import ro.unibuc.hello.data.User.UserDTO;
+import ro.unibuc.hello.data.product.ProductDTO;
+import ro.unibuc.hello.data.product.ProductEntity;
+import ro.unibuc.hello.service.UserService;
+
+import java.util.List;
+
+@Controller
+public class UserController {
+    @Autowired
+    private UserService userService;
+
+    @PostMapping("/api/users")
+    @ResponseBody
+    public void postUser(@RequestBody UserDTO userDTO) {
+        try {
+            userService.saveUser(userDTO);
+        } catch (Exception E) {
```

Comment:
unused variable

Reply:
If I can ignore my IDE I can ignore this comment aswell